charts/v3.0.0/csi-driver-nfs/values.yaml

```diff
@@ -0,0 +1,28 @@
+image:
+    nfs:
+        repository: mcr.microsoft.com/k8s/csi/nfs-csi
```

Comment:
Can we use the kubernetes community repo? k8s.gcr.io/sig-storage/nfsplugin

Reply:
Can we look into fixing it? As a community project we should be using community resources to host.

Reply:
fixed by https://github.com/kubernetes-csi/csi-driver-nfs/pull/195, actually that build pipeline works now